projects/ngrx.io/content/guide/effects/operators.md

```diff
@@ -0,0 +1,176 @@
+# Effects operators
+
+As part of Effect library, NgRx provides some useful operators that are frequently
+used. 
+
+
+## `ofType`
+
+The `ofType` operator filters the stream of actions based on either string
```

Comment:
Should we make clear it's the `type` string?

Reply:
Added